pkgs/development/libraries/gl3w/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, python3, cmake, libglvnd, libGLU }:
+
+stdenv.mkDerivation rec {
+  pname = "gl3w";
+  version = "2022-03-24";
```

It doesn't look like this project has stable releases. (Please update the commit message as well.)

```suggestion
  version = "unstable-2022-03-24";
```

---

Fixed. Thanks for the catch!